draft-irtf-cfrg-opaque.md

```diff
@@ -1677,8 +1680,9 @@ parameters that are needed to prevent cross-protocol or downgrade attacks.
 Absent an application-specific profile, the following configurations are RECOMMENDED:
 
 - ristretto255-SHA512, HKDF-SHA-512, HMAC-SHA-512, SHA-512,
-    Argon2id(t=1, p=4, m=2^21), ristretto255
-- P256-SHA256, HKDF-SHA-256, HMAC-SHA-256, SHA-256, Argon2id(t=1, p=4, m=2^21), P-256
+    Argon2id(S = zeroes(16), p = 4, T = 32, m = 2^21, t = 1, v = 0x13, K = nil, X = "RFCXXXX", y = 2), ristretto255
+- P256-SHA256, HKDF-SHA-256, HMAC-SHA-256, SHA-256,
+    Argon2id(S = zeroes(16), p = 4, T = 32, m = 2^21, t = 1, v = 0x13, K = nil, X = "RFCXXXX", y = 2), P-256
```

Comment:
That would add another point: associated data.

Reply:
X is the associated data here.

Reply:
Yes, what I meant to say is that this would another parameter that is not specified by the spec which will have to be silently agreed on to make implementations compatible.

Basically, until now associated data was left empty (I didn't check more implementations, but kevinlewi's and stef's is empty), now, implementations will probably use `RFCXXXX` as their associated data for all KSF's, even though the spec doesn't say you should do that.